bom/runtime/pom.xml

```diff
@@ -863,6 +865,39 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.datastax.oss</groupId>
+                <artifactId>java-driver-core</artifactId>
+                <version>${datastax.java.driver.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.apache.tinkerpop</groupId>
+                        <artifactId>gremlin-core</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.tinkerpop</groupId>
+                        <artifactId>gremlin-driver</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.tinkerpop</groupId>
+                        <artifactId>tinkergraph-gremlin</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.ow2.asm</groupId>
+                        <artifactId>asm-util</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>com.datastax.oss</groupId>
+                <artifactId>java-driver-mapper-runtime</artifactId>
+                <version>${datastax.java.driver.version}</version>
+            </dependency>
+            <dependency>
```

Comment:
Do we really need this one at runtime? This is typically only needed at compile-time (to generate code).

Reply:
This is used only in `integration-tests` module and somehow integration tests module has access to dependencies that are in this `runtime/pom.xml`